README.rst

```diff
@@ -214,8 +214,8 @@ Choose from the list of available rules:
 
   Configuration options:
 
-  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array
-    syntax; defaults to ``'long'``
+  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array syntax;
```

Comment:
I didn't change any line length. I expected the README file to not change at all but actually got surprised the related test failed when I ran the test suite.

In `fix --help` output, as `` ` `` characters are replaced by color tags and keywords become shorter so wordwrapping is impacted and sometimes allows a little more text before a newline. As the README file uses the newlines from the `fix --help` output without changing them, it is impacted as well. Not sure we can do something about it or if it's even worth it.